indexer/services/comlink/src/controllers/api/v4/addresses-controller.ts

```diff
@@ -499,9 +500,29 @@ router.post(
   ExportResponseCodeStats({ controllerName }),
   async (req: express.Request, res: express.Response) => {
     const start: number = Date.now();
-    const { address, token, language = 'en' } = matchedData(req) as RegisterTokenRequest;
+    const {
+      address, token, language = 'en', timestamp, message, signedMessage, pubKey, walletIsKeplr,
+    } = matchedData(req) as RegisterTokenRequest;
 
     try {
+      const failedValidationResponse = walletIsKeplr
+        ? validateSignatureKeplr(
+          res, address, message, signedMessage, pubKey,
+        )
+        : await validateSignature(
+          res,
+          AccountVerificationRequiredAction.REGISTER_TOKEN,
+          address,
+          timestamp,
+          message,
+          signedMessage,
+          pubKey,
+          '',
+        );
```

Comment:
this is a little complicated for inline, can we just make this into an if statement?